scripts/install_sonobuoy.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+# Usage: sh mixedos_sonobuoy.sh runs with default values
+# Usage: sh mixedos_sonobouy.sh install 0.56.17 arm64
+# Usage: sh mixedos_sonobouy.sh delete
+
+action=${1:-install}
+version=${2:-0.57.2}
+arch=${3:-amd64}
+
+# adopt golang error handling in bash check variables are passed in appropriately - if not return appropriate error message
+
+installation(){
+    echo "Installing sonobuoy version ${version} for mixedos validation"
```

Comment:
can you separate this on another func that this is needed only for mixedos validation... may be add a parameter to be passed in